### PR TITLE
add older IDB versions and documentation

### PR DESCRIPTION
**Pull request by nicHoch**

### CODE REVIEW

File: docs/idb_versioning.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
+IDB Versioning
+==============
+
+After a new IDB Version has been released it should be added to stixcore. All awailable IDB versions to stixcore are managed in the file `stixcore\data\idb\idbVersionHistory.json`
```
Comment (samaloney) [inline docs/idb_versioning.rst:4]:
```suggestion
After a new IDB Version has been released it should be added to stixcore. All available IDB versions to stixcore are managed in the file `stixcore\data\idb\idbVersionHistory.json`
```

**Review by samaloney — APPROVED** [review]
Looks good. 

I wonder if we need to handle multiple periods for the same version? E.g 167 has to valid time ranges it was run once on the nominal side and once on the redundant side. Could be important if we want to use the same approach other things.

**Comment by nicHoch** [comment]
the current code-base should support multiple time periods for the same IDB version as long as they do not overlap.

**Comment by nicHoch** [comment]
fix: https://github.com/i4Ds/STIXCore/issues/140

**Pull request merged**